[Bug] unable to run food-delivery example

### What are you really trying to do?

Start the example in the `food-delivery` folder using instructions provided in the related `README.md` file.

### Describe the bug

The `npm run dev` command in the `food-delivery` folder display some errors.

This an excerpt of the `npm run dev` command output:

```
worker:dev: /Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:859
worker:dev:     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
worker:dev:            ^
worker:dev: TSError: ⨯ Unable to compile TypeScript:
worker:dev: src/worker.ts(4,49): error TS2307: Cannot find module 'common/lib/temporal-connection' or its corresponding type declarations.
worker:dev:
worker:dev:     at createTSError (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:859:12)
worker:dev:     at reportTSError (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:863:19)
worker:dev:     at getOutput (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:1077:36)
worker:dev:     at Object.compile (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:1433:41)
worker:dev:     at Module.m._compile (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:1617:30)
worker:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1293:10)
worker:dev:     at Object.require.extensions.<computed> [as .ts] (/Users/flerro/Labs/temporal/samples-typescript/food-delivery/node_modules/ts-node/src/index.ts:1621:12)
worker:dev:     at Module.load (node:internal/modules/cjs/loader:1096:32)
worker:dev:     at Function.Module._load (node:internal/modules/cjs/loader:935:12)
worker:dev:     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:84:12) {
worker:dev:   diagnosticCodes: [ 2307 ]

### Minimal Reproduction

1. Clone the repository 
2. Move into the `food-delivery` directory
3. Install the dependencies via `npm`
4. Run the example via `npm run dev`

### Environment/Versions

- OS and processor: M1 Mac

--- a/food-delivery/.post-create
+++ b/food-delivery/.post-create
@@ -14,7 +14,6 @@ Use Node version 16+:
 
 Then, in the project directory, run:
 
-  {cyan npm run build}
   {cyan npm run dev}
 
 And visit:

--- a/food-delivery/turbo.json
+++ b/food-delivery/turbo.json
@@ -10,7 +10,9 @@
       "outputs": []
     },
     "dev": {
-      "cache": false
+      "dependsOn": ["build"],
+      "cache": false,
+      "persistent": true
     },
     "start": {},
     "worker#build": {

